slack.py

```diff
@@ -58,16 +58,12 @@ def _filter_new_messages(self, messages):
             self.logger.debug(f"Checking message with timestamp: {ts}")
 
             replies = self.client.conversations_replies(channel=self.channel_id, ts=ts)
-            if (
-                self.last_seen_timestamp is None
-                or float(ts) > float(self.last_seen_timestamp)
-            ) and len(replies["messages"]) == 1:
```

Comment:
We will still this condition `len(replies["messages"]) == 1` because we want to only consider messages that doesn't have any replies already.

Reply:
I get why we had len(replies["messages"]) == 1 to only pick messages without replies but it actually makes us miss some important failure messages. For example if the team already replied before Bugzooka got to it or if Bugzooka restarts and finds old failure messages that already have replies.. it won’t catch those.

For now, I’d rather reply to all failure messages to make sure nothing slips through

Later on we can improve this by having Bugzooka check if Jedi has already replied, so it only skips those threads where it’s already active.

Reply:
Only thing that is redundant here is, if the bot has previously replied, it will shoot another reply which looks ugly. Keeping the current approach can we do a reply only when the bot hasn't responded before?